pkg/harvester-manager/machine-config/harvester.vue

```diff
@@ -1117,9 +1098,7 @@ export default {
     },
 
     vGpuOptionLabel(opt) {
-      const vGpu = this.vGpuEnabledDevices[opt];
-
-      return `${ vGpu?.type?.replace(VGPU_PREFIX.NVIDIA, '') } - ${ vGpu?.id } (allocatable: ${ this.vGpusAllocatable[vGpu?.type] })`;
+      return opt;
```

Comment:
`allocatable` count is meanful to user, do you think we can display allocatable count in edit / create mode ?

```
 const vGpuDevice = Object.values(this.vGpuDevices).find((d) => d.type === opt);

      return vGpuDevice && vGpuDevice.allocatable > 0 & (this.mode === _CREATE || this.mode === _EDIT) ? `${ opt } (allocatable: ${ vGpuDevice.allocatable })` : opt;
```

Reply:
Yeah let's add it, but i'm not sure if that number will be always `1` for each vGPU. @ibrokethecloud can you confirm ?

Reply:
`allocatable` count is what we assigned for each vGPU type in L726.
 
The vGPU type count retrieved from `/v1/harvester/clusters/local?link=deviceCapacity` is the total available vGPU for each type, right? 

<img width="1496" alt="Screenshot 2024-08-09 at 12 08 03 PM" src="https://github.com/user-attachments/assets/2f8cf998-9a5d-4b01-80ce-4630b622d27c">